driver-core/src/test/functional/com/mongodb/client/model/expressions/ArrayExpressionsFunctionalTest.java

```diff
@@ -17,6 +17,7 @@
 package com.mongodb.client.model.expressions;
 
 import com.mongodb.MongoCommandException;
+import org.bson.Document;
```

Comment:
[I cannot leave the comment on the relevant line as the line came from a different PR]

Now that we have `isIntegerOr`, let's replace `array123.elementAt((IntegerExpression) of(0.0))` in `ArrayExpressionsFunctionalTest.elementAtTest` with `array123.elementAt(of(0.0).isIntegerOr(of(-1)))`. This comment is from a [previous PR](https://github.com/mongodb/mongo-java-driver/pull/1050#discussion_r1034061470).